src/client/components/MealsComponent/MealsList.jsx

```diff
@@ -0,0 +1,49 @@
+import React, { useState, useEffect } from 'react';
+import Meal from './Meal';
+
+export default function MealsList() {
+  const [meals, setMeals] = useState([]);
+  const [error, setError] = useState(null);
+
+  useEffect(() => {
+    let isMounted = true; // track whether component is mounted
```

Comment:
This is generally not needed, did you run into some specific problem that made you add this?